js/modules/k6/experimental/csv/data.csv

```diff
@@ -0,0 +1,21 @@
+firstname,email
```

Comment:
Nit; but do we really need to have non-ASCII characters, IDNs and... are there email addresses with whitespaces? 🤔 😅

Reply:
Hehe I used one of my tools to produce "fake" csv, I'm completely open to using another file 😅
This should probably live in a test case anyway 👍🏻 I'll remove the file ^^

Reply:
To be honest, I'm fine with having this file with fake data, I was just captured by some of details of the data shape haha 😄

Reply:
My little tool tries to be inclusive ^^